timescaledb/how-to-guides/continuous-aggregates/index.md

```diff
@@ -20,6 +21,7 @@ only the data that has changed needs to be computed, not the entire dataset.
 *   [Manage materialized hypertables][cagg-mat-hypertables].
 *   [Use real-time aggregates][cagg-realtime].
 *   [Compression with continuous aggregates][cagg-compression].
+*   [Migrate your continuous aggregates][cagg-migrate] from old to new format.
```

Comment:
Should we be more specific and say `old (pre TimescaleDB 2.7)` ?

Reply:
```suggestion
*   [Migrate your continuous aggregates][cagg-migrate] from old to new format.
    Continuous aggregates created in TimescaleDB 2.7 and above are in the new
    format, unless explicitly created in the old format.
```